boards/arm/mec172xevb_assy6906/mec172xevb_assy6906.dts

```diff
@@ -76,14 +77,34 @@
 &uart1 {
 	status = "okay";
 	current-speed = <115200>;
+	pinctrl-0 = <&pinctrl_uart1_tx &pinctrl_uart1_rx>;
+	pinctrl-names = "default";
 };
 
 &adc0 {
 	status = "okay";
+	pinctrl-0 = <&pinctrl_adc0_in0 &pinctrl_adc0_in3
+		     &pinctrl_adc0_in4 &pinctrl_adc0_in5>;
+	pinctrl-names = "default";
 };
 
 &espi0 {
 	status = "okay";
+	pinctrl-0 = <&pinctrl_espi_reset &pinctrl_espi_cs
+		     &pinctrl_espi_alert &pinctrl_espi_clk
+		     &pinctrl_espi_io0 &pinctrl_espi_io1
+		     &pinctrl_espi_io2 &pinctrl_espi_io3>;
+	pinctrl-names = "default";
+};
+
+/* If ESPI_CS is open drain */
+&pinctrl_espi_cs {
+	drive-open-drain;
```

Comment:
@jvasanth1 I recall from MEC152x that this is not required. Is this different in MEC172x? 

This since the eSPI Host can select either push-pull or open-drain for this pin during eSPI bus enumeration the MCHP ESPI HW block must be able to dynamically change the pin’s output buffer type.  
This is not reflected in the GPIO control register.  This is one of the very few pins that has a pin feature automatically configured.

Reply:
Yes, this is not required for the two pins (ESPI_CS and ESPI_ALERT), will remove from the board .dts